buildpipeline/DotNet-CoreFx-Trusted-Windows-NoTest.json

```diff
@@ -17,7 +16,7 @@
         "targetType": "inline",
         "filePath": "",
         "arguments": "",
-        "script": "if (Test-Path \"$(AgentToolsPath)\\Begin.ps1\") {\n     \"Begin.ps1 script found. Executing...\"\n     cd $(AgentToolsPath)\n     & $(AgentToolsPath)\\Begin.ps1\n} else {\n   \"Begin.ps1 script does not exist. Moving on...\"\n}",
+        "script": "if (Test-Path \"$(AgentToolsPath)\\begin.ps1\") {\n   \"$(AgentToolsPath)\\begin.ps1 script found. Executing...\"\n    $(AgentToolsPath)\\begin.ps1\n} else {\n   \"$(AgentToolsPath)\\begin.ps1 script does not exist. Moving on...\"\n}",
```

Comment:
$(AgentToolsPath) used here not $AGENTTOOLSPATH

Reply:
Are you suggesting we use env variables here? If so then you would need to do `$env:AGENTTOOLSPATH` but I'm not sure I would suggest that, if we can use the VSTS variable I would suggest using that.

Reply:
No, I was just pointing out that in this file we used $(AgentToolsPath) compared to $AGENTTOOLSPATH which makes sense since this is a Windows build def. I just thought we would use $(AgentToolsPath) anywhere regardless of the OS